### ISSUE TITLE
Feature: Update check

### ISSUE THREAD

**Issue by wheresvic** [zellij-org/zellij#439]
It would be great if zellij could automatically check if there is an update an notify the user if yes.

**Comment by imsnif** [comment]
Tbh I'm a little cautious about doing such things. I get a little uneasy when executables I download to my machine, which should have nothing on their own to do with the Internet, start sending http requests to random places... how about if we make a `--check-for-updates` flag instead?

**Comment by wheresvic** [comment]
Yep that works as well! I could then make a script that defaults zellij to run with that flag set :)
